config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -75,13 +74,6 @@ periodics:
           cpu: 1000m
           memory: 3Gi
   cluster: k8s-infra-prow-build
-  resources: &id001
-    requests:
-      cpu: 2000m
-      memory: 6Gi
-    limits:
-      cpu: 2000m
-      memory: 6Gi
```

Comment:
why are these resources deleted?

Reply:
if you see above it they were assigned twice,..

Reply:
the same resource limits are still assigned

Reply:
I can't see them added in other place, are no longer needed?

Reply:
oh sorry I was thinking of a different file...the resources here arnt removed they are indented....view the full change

Reply:
the full change only shows lines supressions :thinking: 

@cpanato this file seems generated 

>   - generated   # AUTO-GENERATED by releng/generate_tests.py - DO NOT EDIT!

can you please check this has been correctly generated?

Reply:
if this is generated it means something is wrong on the source, let's identify it and fix it or open a new issue to follow up but not forget about it